blocks/headline-block/index.story.jsx

```diff
@@ -10,9 +10,9 @@ export default {
 export const customHeadline = () => {
   const headlineString = text('headlineString', 'Man Bites Dog');
   // if we want to allow custom fonts https://stackoverflow.com/a/63128475/7491536
-  const primaryFont = text('primaryFont (web-safe)', 'Arial');
+  // const primaryFont = text('primaryFont (web-safe)', 'Arial');
```

Comment:
could probably delete this commented-out line. it's a fun demo though to be able to change the font live